backend/models/Session.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
```

Comment:
Also, regarding package structure, could you potentially do mongo/models/*.js.
Just sot here is no confusion with the domain models. 🙃

Reply:
Fixed.